lib/whitehall_importer/create_edition.rb

```diff
@@ -2,15 +2,15 @@
 
 module WhitehallImporter
   class CreateEdition
-    attr_reader :document, :whitehall_document, :whitehall_edition, :edition_number, :user_ids
+    attr_reader :document, :current, :whitehall_edition, :edition_number, :user_ids
 
     def self.call(*args)
       new(*args).call
     end
 
-    def initialize(document, whitehall_document, whitehall_edition, edition_number, user_ids)
+    def initialize(document:, current:, whitehall_edition:, edition_number:, user_ids:)
```

Comment:
Should we make some of these arguments optional (e.g current: true, edition_number: 1 user_ids: {}) just to make testing less verbose?

Reply:
sure